### PR TITLE
SEO updates to /managed/kafka

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- SEO updates to /managed/kafka
- Add webinar

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed/kafka
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/15c89-PyMc-QnSHglG1hf_Tub5lMkEcVYALL0rm5-LvA/edit#) and resolve comments

## Issue / Card

Fixes #2782

### CODE REVIEW

File: templates/managed/kafka.html [hunk 1]
```diff
@@ -154,151 +163,182 @@ <h3>Cloud architecture</h3>
...
-        <td><strong>Hybrid-cloud</strong></td>
-        <td colspan="2">Persistent streaming between all cloud types and management across all</td>
-      </tr>
-      <tr>
-        <td><strong>Multi-cloud</strong></td>
-        <td colspan="2">Persistent streaming between all cloud types and management across all</td>
-      </tr>
-    </table>
+        <td colspan="2">Any OpenStack, bare metal and key third-party providers such as VMWare.<br />
+          We can support your specific cloud provider - <a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>.</td>
```
Comment (anthonydillon) [inline templates/managed/kafka.html:167]: I don't think this one should be a button. Would be more consistent as a link.

File: templates/managed/kafka.html [hunk 2]
```diff
@@ -154,151 +163,182 @@ <h3>Cloud architecture</h3>
...
+          <td colspan="2">Yes &mdash; Canonical&rsquo;s Kafka experts cover all geographic regions and time-zones</td>
+        </tr>
+        <tr>
+          <td><strong>Break/Fix response</strong></td>
+          <td colspan="2">Yes</td>
+        </tr>
+        <tr>
+          <td><strong>SLA</strong></td>
+          <td colspan="2">Yes, 99.9% up-time.<br />
+            We can support your specific requirements - <a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>.</td>
```
Comment (anthonydillon) [inline templates/managed/kafka.html:276]: Here too

File: templates/managed/kafka.html (hunk 1 above)
Comment (pmahnke) [reply templates/managed/kafka.html]: you are right, moving too fast!

File: templates/managed/kafka.html (hunk 2 above)
Comment (pmahnke) [reply templates/managed/kafka.html]: fixed

Comment (anthonydillon) [reply templates/managed/kafka.html]: Sorry, but I think they should be lowercased.

File: templates/managed/kafka.html [hunk 3]
```diff
@@ -154,151 +163,182 @@ <h3>Cloud architecture</h3>
...
+          <td colspan="2">Yes &mdash; Canonical&rsquo;s Kafka experts cover all geographic regions and time-zones</td>
+        </tr>
+        <tr>
+          <td><strong>Break/Fix response</strong></td>
+          <td colspan="2">Yes</td>
+        </tr>
+        <tr>
+          <td><strong>SLA</strong></td>
+          <td colspan="2">Yes, 99.9% up-time.<br />
+            We can support your specific requirements - <a href="/managed/contact-us" class="js-invoke-modal">Contact us</a>.</td>
```
Comment (anthonydillon) [inline templates/managed/kafka.html:276]:
```suggestion
            We can support your specific requirements - <a href="/managed/contact-us" class="js-invoke-modal">contact us</a>.</td>
```

File: templates/managed/kafka.html [hunk 4]
```diff
@@ -154,151 +163,182 @@ <h3>Cloud architecture</h3>
...
-        <td><strong>Hybrid-cloud</strong></td>
-        <td colspan="2">Persistent streaming between all cloud types and management across all</td>
-      </tr>
-      <tr>
-        <td><strong>Multi-cloud</strong></td>
-        <td colspan="2">Persistent streaming between all cloud types and management across all</td>
-      </tr>
-    </table>
+        <td colspan="2">Any OpenStack, bare metal and key third-party providers such as VMWare.<br />
+          We can support your specific cloud provider - <a href="/managed/contact-us" class="js-invoke-modal">Contact us</a>.</td>
```
Comment (anthonydillon) [inline templates/managed/kafka.html:167]:
```suggestion
          We can support your specific cloud provider - <a href="/managed/contact-us" class="js-invoke-modal">contact us</a>.</td>
```

File: templates/managed/kafka.html (hunk 2 above)
Comment (pmahnke) [reply templates/managed/kafka.html]: done.

**Pull request merged**